store/sqlstore/token_store.go

```diff
@@ -68,6 +68,22 @@ func (s SqlTokenStore) GetByToken(tokenString string) store.StoreChannel {
 	})
 }
 
+func (s SqlTokenStore) GetByTypeAndExtra(tokenType, extraString string) store.StoreChannel {
```

Comment:
There are no indexes on Type and Extra. We should probably add them if we are selecting by those.